app/experimenter/nimbus-ui/src/components/PageHome/index.tsx

```diff
@@ -106,4 +116,17 @@ const PageHome: React.FunctionComponent<PageHomeProps> = () => {
   );
 };
 
+const RefetchAlert = () => (
+  <Alert variant="warning" data-testid="refetch-alert" className="mt-4">
+    <Alert.Heading>Error fetching experiments</Alert.Heading>
+    <p>
+      An error occured. This usually happens when Experimenter is mid-deploy.
+    </p>
+    <p>
+      Fetching experiments will be retried one time automatically in{" "}
+      {REFETCH_DELAY / 1000} seconds.
+    </p>
+  </Alert>
```

Comment:
Can we reuse this component?

Reply:
Now that I'm going to add this to `App/index.tsx` we can. With the heading, body, and alert `variant` differing, I didn't share it with `ApolloErrorAlert` or `AppErrorBoundary` alert.